oak_containers/proto/interfaces.proto

```diff
@@ -79,6 +97,15 @@ service Launcher {
   rpc NotifyAppReady(google.protobuf.Empty) returns (google.protobuf.Empty) {}
 }
 
+// Defines the service exposed by the orchestrator, that can be invoked by the application.
+service ControlPlane {
+  // Request a new enclave encryption key as part of Key Provisioning.
+  rpc GetEncryptionKey(GetEncryptionKeyRequest) returns (GetEncryptionKeyResponse) {}
+
+  // Replace old enclave encryption key with a new one as part of Key Provisioning.
+  rpc SendEncryptionKey(SendEncryptionKeyRequest) returns (google.protobuf.Empty) {}
```

Comment:
Perhaps call this something more generic, e.g. `ProvisionSecret` or something like that? in principle, we may want to provision other kinds of things in the future (e.g. symmetric storage keys, or signing keys, or secrets for deriving other keys, etc. -- they all have in common that they are secrets).

Reply:
Renaming to `ProvisionSecrets`

Reply:
In my suggestion, `Provision` is the verb, so `SendProvision` is not grammatically correct. Or did you mean something like `Provisioning`?

Reply:
I think somehow the incorrect version was submitted? At least, IMO `SendProvisionSecrets` is not grammatically correct (what is a "provision secret"?)